### ISSUE TITLE
Lack of LDAP/M$ AD Authentication

### ISSUE THREAD

**Issue by gnu4cn** [avored/avored-rust-cms#210]
If LDAP/M$ AD authentication deployed, Avored will be more useful for mass adoption.

Is it possible to integrate/make this feature?

**Comment by indpurvesh** [comment]
@gnu4cn Yes definately. I know it much important it is for any company to have an LDAP login as a feature to easy role/permission access based on groups and etc so yes for sure will implement the feature.

**Comment by indpurvesh** [comment]
@gnu4cn 

Just to let you know that we did implemented with LDAP in avored. It is not ready to tested it out yet. Few more improvments needs to done. will create a new feature/issue but will close this one
